http/vibe/http/server.d

```diff
@@ -795,20 +803,33 @@ final class HTTPServerRequest : HTTPRequest {
 			The fields are stored in the same order as they are received.
 
 			Remarks:
-				This field is only set if HTTPServerOption.parseFormBody is set.
-
 				A form request must either have the Content-Type
 				"application/x-www-form-urlencoded" or "multipart/form-data".
 		*/
-		FormFields form;
+		@property ref const(FormFields) form() @safe {
+			if (_form.isNull)
+				parseFormAndFiles;
 
-		/** Contains information about any uploaded file for a HTML _form request.
+			return _form.get;
+		}
 
-			Remarks:
-				This field is only set if HTTPServerOption.parseFormBody is set
-				and if the Content-Type is "multipart/form-data".
+		private Nullable!FormFields _form;
+
+		private void parseFormAndFiles() @safe {
+			parseFormData(_form, _files, contentType, bodyReader, MaxHTTPHeaderLineLength);
```

Comment:
If I'm not overlooking something, I think this needs to set `_form` to `.init` first, or it would try to access the null field.